plugins/database/postgresql/postgresql.go

```diff
@@ -338,6 +338,17 @@ func (p *PostgreSQL) customDeleteUser(ctx context.Context, username string, revo
 	}()
 
 	for _, stmt := range revocationStmts {
+		if containsMultilineStatement(stmt) {
+			// Execute it as-is.
+			m := map[string]string{
+				"name":     username,
+				"username": username,
+			}
+			if err := dbtxn.ExecuteTxQueryDirect(ctx, tx, m, stmt); err != nil {
+				return err
```

Comment:
Do you think it'd be a good idea to wrap this error so it's labeled as a multiline query that was attempted?

`return fmt.Errorf("failed to execute multiline query: %w", err)`

Reply:
I think I will leave this one as is since we already aren't doing it for creation